plugins/cros-ec/fu-cros-ec-usb-device.c

```diff
@@ -764,6 +764,29 @@ fu_cros_ec_usb_device_jump_to_rw(FuDevice *device)
 	return TRUE;
 }
 
+static gboolean
+fu_cros_ec_usb_device_stay_in_ro(FuDevice *device, GError **error)
+{
+	gsize response_size = 1;
+	guint8 response;
+	guint16 subcommand = UPDATE_EXTRA_CMD_STAY_IN_RO;
```

Comment:
does this cmd exist in all FW versions?  or do you need some guards for it?

Reply:
Right; that was my concern if it's called from a second location if any extra guards are going to be needed from that second location calling.